measures/measures-schema.yaml

```diff
@@ -197,3 +205,55 @@ definitions:
       - thoracicSurgery
       - urology
       - vascularSurgery
+
+  eligibilityOption:
+    type: object
+    properties:
+      submissionMethods: { $ref: #/definitions/qualityCodesSubmissionMethods }
+      sexCode:
+        enum: [M, F]
```

Comment:
I was thinking if it is null that key/value would be omitted entirely. Is the reason to include the `keyName: null` for consistency or another reason? It will bloat the file a lot - there will be a lot of null sexCodes, minAge and maxAge.